src/main/java/subway/application/sectionReader/SectionReader.java

```diff
@@ -0,0 +1,24 @@
+package subway.application.sectionReader;
```

Comment:
```suggestion
package subway.application.sectionreader;
```

<br/>

<img width="588" alt="image" src="https://github.com/woowacourse/jwp-subway-path/assets/50176238/e332c5e1-2093-43fe-b42e-61dcb283d344">

Reply:
[참고](https://codedragon.tistory.com/228)해주세요 ㅎㅎ